cookbooks/steinberger/doc/steinberger.tex

```diff
@@ -0,0 +1,157 @@
+$T$ is temperature, $T_\text{ref}$ is the reference temperature profile, 
+and $\Delta T$ is the deviation from the reference temperature profile. 
+
+This reference profile can be chosen in two different ways: On the one hand, it can be chosen as the laterally averaged temperature (and in this case, a number of depth slices for this lateral averaging can be specified as  well). This is the original formulation of \cite{stca06}, and the default of the material model. 
+On the other hand, the adiabatic temperature profile can be chosen as the reference. 
+However, the radial profile needs to be adapted based on how this referene temperature is chosen. 
+If the reference profile uses the laterally averaged temperature, then the radial profile needs to include a high viscosity in the lithosphere (where it is cold), and a low viscosity near the core-mantle boundary (where it is warm). If the reference profile is the adiabatic profile, then the temperature will deviate from this reference in the top and bottom thermal boundary layers already, leading to changes in viscosity. So in this case, the radial profile should not include these boundary layers (because otherwise we would compute their effect twice). This option allows the viscosity in the boundary layers to develop based on the temperature in the model, which is why we choose it for this cookbook. 
+
+The default data directory already contains two radial viscosity files, one for each of these cases. 
+The file \url{data/material-model/steinberger/radial-visc.txt} is the original Steinberger and Calderwood \cite{stca06} profile (with an interpolation between the original discrete layers) and for use with the laterally averaged temperature. The file \url{data/material-model/steinberger/radial-visc-simple.txt} is for use with the adiabatic profile. To illustrate the difference, the content of both files is plotted in Figure~\ref{fig:steinberger-viscosity}. 
+
+\begin{figure}
+  \includegraphics[width=0.48\textwidth]{cookbooks/steinberger/doc/radial-visc.pdf}
+  \includegraphics[width=0.48\textwidth]{cookbooks/steinberger/doc/radial-visc-simple.pdf}
+  \caption{\it Left: Viscosity profile based on the original \cite{stca06} formulation, intended for use with a temperature-dependence of viscosity based on the laterally averaged temperature. Right: Modified viscosity profile without boundary layers, intended for use with a temperature-dependence of viscosity based on an adiabatic temperature profile.}
+  \label{fig:steinberger-viscosity}
+\end{figure}
+
+In order to improve solver convergence, the material model has additional parameters that allow it to limit the viscosity variations. 
+Because of the resolution in this cookbook we limit the overall viscosity
+between $10^{20}$~Pa~s and $5 \times 10^{23}$~Pa~s and the lateral viscosity variations to 
+three orders of magnitude in both directions (for a total of six orders of
+magnitude. This allows the features of the flow field to be resolved. 
```

Comment:
I think I don't understand this. If you limit the overall viscosity to 1e20...1e23, how can the lateral variation be in a larger range? Or maybe I don't understand the word "overall" :-)